Security/Firewall/GetJWTListener.php

```diff
@@ -98,17 +99,27 @@ public function handle(GetResponseEvent $event)
     {
         $request = $event->getRequest();
 
-        if ($this->options['post_only'] && !$request->isMethod('POST')) {
-            $event->setResponse(new JsonResponse('invalid method', 405));
-            return;
+        $is_json_request = $request->headers->get('content_type') === 'application/json';
```

Comment:
And Yoda conditions...

Anyway, there is no need for this variable because the condition can be rewritten as simply:

```php
if ('json' === $request->getContentType()) {
```